buildkite/schedule_and_publish/pipeline.yml

```diff
@@ -40,27 +40,11 @@ steps:
 
   - wait
 
-  - label: "Publish Benchmark Results on Slack"
+  - label: "Publish to Pull Requests"
```

Comment:
I'm fine to leave this as is, but it would be nice to link to #98 or otherwise comment that this is a temporary change / we will eventually be posting to slack when things go wrong again, just not _right now_.

Reply:
Cool, added a comment (and fixed the build) but GitHub is having issues so the commit isn't showing up here yet.